newdle/client/src/components/TopHeader.js

```diff
@@ -0,0 +1,54 @@
+import React, {useState} from 'react';
+import {Grid, Header} from 'semantic-ui-react';
+import {Route, Switch, Link} from 'react-router-dom';
+import Navigator from './Navigator';
+import UserMenu from './UserMenu';
+
+import styles from './TopHeader.module.scss';
+
+const steps = [
+  {
+    title: 'Choose the Participants',
+    description: 'You can also "Skip" this step and add them later',
+  },
+  {
+    title: 'Choose the Timeslots',
+    description: 'Which will be presented as options to the participants',
+  },
+  {
+    title: 'Finalize',
+    description: 'Do something final here',
+  },
+];
+
+export default function TopHeader() {
+  const [active] = useState(1);
+  return (
+    <header className={styles.header}>
+      <Grid columns={3}>
+        <Grid.Column>
+          <Header as="h1" className={styles.title}>
+            <Link to="/">newdle</Link>
+          </Header>
+        </Grid.Column>
+        <Grid.Column />
+        <Grid.Column textAlign="right">
+          <UserMenu />
+        </Grid.Column>
+        <Grid.Row centered>
+          <Grid.Column />
+          <Grid.Column>
+            <Switch>
+              <Route
+                exact
+                path="/new"
+                component={() => <Navigator steps={steps} active={active} />}
```

Comment:
That's not the correct way to use `component`. It should be something like `component={Navigatorr}`, but since you need to specify props here you want `render={() => <Navigator .. />}`